inference-engine/src/mkldnn_plugin/nodes/mkldnn_ada_pooling.cpp

```diff
@@ -0,0 +1,361 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "mkldnn_ada_pooling.h"
```

Comment:
Please rename files to more obvious like mkldnn_adaptive_pooling

Reply:
Done